ServiceRequest/ServiceRequest-NonWGSTestOrderForm-HaemOncology-Example.json

```diff
@@ -0,0 +1,90 @@
+{
+  "resourceType": "ServiceRequest",
+  "id": "ServiceRequest-NonWGSTestOrderForm-HaemOncology-Example",
+  "extension": [
+    {
+      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-Coverage",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-FundingCategory",
+            "code": "nhs",
+            "display": "NHS"
+          }
+        ]
+      }
+    }
+  ],
+  "status": "active",
+  "intent": "order",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-GenomeSequencingCategory",
+          "code": "cancer-non-wgs",
+          "display": "Cancer - Non-WGS"
+        }
+      ]
+    }
+  ],
+  "priority": "routine",
+  "code": {
+    "coding": [
+      {
+        "system": "https://fhir.nhs.uk/CodeSystem/England-GenomicTestDirectory",
+        "code": "M91",
+        "display": "Acute Lymphoblastic Leukaemia"
+      }
+    ]
+  },
+  "subject": {
+    "reference": "Patient/Patient-PatrickSammy-Example",
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449307873"
+    }
+  },
+  "authoredOn": "2023-11-13",
+  "requester": {
+    "reference": "PractitionerRole/PractitionerRole-HazelSmithPathology-Example"
+  },
+  "reasonCode": [
+    {
+      "coding": [
+        {
+          "system": "http://snomed.info/sct",
+          "code": "49401003",
+          "display": "Bone marrow aspiration procedure"
+        }
+      ]
+    }
+  ],
+  "supportingInfo": [
+    {
+      "reference": "Observation/Observation-NoTransplant-Example"
+    },
+    {
+      "reference": "Observation/Observation-AcuteLyphoblasticLeukamia-Example"
+    },
+    {
+      "reference": "Condition/Condition-Bruising-Example"
+    },
+    {
+      "reference": "Condition/Condition-Hepatosplenomegaly-Example"
+    }
+  ],
+  "specimen": [
+    {
+      "reference": "Specimen-BoneMarrowAspiration-Example"
+    }
+  ],
+  "note": [
+    {
+      "text": "No family history of genomic testing"
+    },
+    {
+      "text": "Condition-Low counts: Hb: 83; WBC: 11.9; Neut: 0.51, Plts: 100"
```

Comment:
These could potentially be split out into observation resources, e.g. Heamoglobin, white blood cell count, neutrophil count and platelet count

Reply:
I changed it to provisional, to ensure that the example conforms to the Condition profile. Cardinality of 1:1 on Condition.verificationStatus.